sdk/identity/azure-identity/src/main/java/com/azure/identity/SharedTokenCacheCredentialBuilder.java

```diff
@@ -8,21 +8,9 @@
  *
  * @see SharedTokenCacheCredential
  */
-public class SharedTokenCacheCredentialBuilder extends CredentialBuilderBase<ManagedIdentityCredentialBuilder> {
-    private String clientId;
+public class SharedTokenCacheCredentialBuilder extends AadCredentialBuilderBase<SharedTokenCacheCredentialBuilder> {
     private String username;
 
-    /**
-     * Sets the client ID for the application.
-     *
-     * @param clientId The client ID for the application.
-     *
-     * @return The updated SharedTokenCacheCredentialBuilder object.
-     * */
```

Comment:
I am assuming this comes from AadCredentialBuilderBase now.

Reply:
Yes